pilot/pkg/security/authn/v1alpha1/policy_applier.go

```diff
@@ -278,6 +280,29 @@ func convertPolicyToAuthNFilterConfig(policy *authn_v1alpha1.Policy, proxyType m
 	return filterConfig
 }
 
+// setDownstreamTlsContext sets the downstream tls context configuration based on the information passed to it
+func setDownstreamTlsContext(tls *auth.DownstreamTlsContext, tlsServerRootCert string, tlsServerCertChain string, tlsServerKey string, alpnProtocols []string, requireClientCerts *wrappers.BoolValue, subjectAltNames []string) {
```

Comment:
This configuration will be added to inbound filter chain of the server, then should it be called upstream?

Reply:
nit pick - Can we consolidate parameters of the same type like

```
func setDownstreamTlsContext(tls *auth.DownstreamTlsContext, tlsServerRootCert, tlsServerCertChain , tlsServerKey string, alpnProtocols, subjectAltNames []string,  requireClientCerts *wrappers.BoolValue) {
```

Reply:
the inbound filter chain TLS context is used to perform TLS termination of the request that comes from the downstream so setDownstreamTlsContext seems more apt